e2e/testinfra/terraform/common/service_accounts.tf

```diff
@@ -38,6 +38,14 @@ module "e2e-gcr-reader-sa" {
   role = "roles/storage.objectViewer"
 }
 
+module "e2e-metric-writer-sa" {
+  source = "../modules/service_account"
+  gcp_sa_id = "e2e-test-metric-writer"
+  gcp_sa_display_name = "Test Metric Writer"
+  gcp_sa_description = "Service account used to write to Google Cloud Monitoring"
+  role = "roles/monitoring.metricWriter"
```

Comment:
What are the metrics going to be written to? Do any other GCP resources need to be provisioned?

Reply:
They will be written to the cloud monitoring API, no other GCP resources are needed.

Reply:
Does the API need to be enabled? If so, API enablement is currently handled here: https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/e2e/testinfra/terraform/common/services.tf

Reply:
Good point, added monitoring service.